docs/apache-airflow-providers-smtp/connections/smtp.rst

```diff
@@ -0,0 +1,80 @@
+.. Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+ ..   http://www.apache.org/licenses/LICENSE-2.0
+
+ .. Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+
+
+.. _howto/connection:smtp:
+
+SMTP Connection
+===============
+
+The SMTP connection type enables integrations with the SMTP client.
+
+Authenticating to SMTP
+----------------------
+
+Authenticate to the SMTP client with the login and password field.
+Use standard `SMTP authentication
+<https://docs.python.org/3/library/smtplib.html>`_
+
+Default Connection IDs
+----------------------
+
+Hooks, operators, and sensors related to SMTP use ``smtp_default`` by default.
+
+Configuring the Connection
+--------------------------
+
+Login
+    Specify the username used for the SMTP client.
+
+Password
+    Specify the password used for the SMTP client.
+
+Host
+    Specify the SMTP host url.
+
+Port
+    Specify the SMTP port to connect to. The default depends on the whether you use ssl or not.
+
+Extra (optional)
+    Specify the extra parameters (as json dictionary)
+
+    * ``from_email``: The email address from which you want to send the email.
+    * ``disable_ssl``: If set to true, then a non-ssl connection is being used. Default is false. Also note that changing the ssl option also influences the default port being used.
```

Comment:
Not mandatory but would be nice to have it as boolean field in the UI

Reply:
I already added it ([line](https://github.com/apache/airflow/pull/29968/files#diff-263bb81fd2d2743284a2b74545d6764328eb662a09f97ccc13a1126e0f99d8b5R137))